semantic-conventions/src/tests/data/markdown/ref_extends/server.yaml

```diff
@@ -0,0 +1,10 @@
+groups:
+  - id: server
+    type: attribute_group
+    prefix: server
+    brief: 'This document defines semantic conventions for common server attributes.'
+    attributes:
+      - id: address
+        type: string
+        brief: 'Domain name.'
```

Comment:
Would be nice if the "briefs" contained where in the hierarchy they are, so it's easier to understand the expect/actual results. like `(Grandparent) domain name` or smt similar.